web/tests/narrow.test.js

```diff
@@ -100,13 +100,6 @@ run_test("empty_narrow_html", ({mock_template}) => {
         actual_html,
         `<div class="empty_feed_notice">
     <h4> This is a title </h4>
-    <div>
-        Some common words were excluded from your search. <br/>You searched for:
-        <span>stream: new</span>
-        <span>topic: test</span>
-            <span>search</span>
-            <del>a</del>
-    </div>
```

Comment:
Am I correct that this is removing our display of stop words that you searched for? That's an important feature to avoid search results being confusing that I suspect @terpimost was not aware of; I don't think removing that feature is an intentional design choice here.